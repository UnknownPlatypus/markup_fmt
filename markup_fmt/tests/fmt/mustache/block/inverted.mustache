{{^ boolean    }}={{/    boolean }}

{{^bool}}
* first
{{/bool}}
* {{two}}
{{^bool}}
* third
{{/bool}}

{{^bool}}B {{^bool}}C{{/bool}} D{{/bool}}
