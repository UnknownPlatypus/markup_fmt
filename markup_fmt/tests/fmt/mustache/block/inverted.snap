---
source: markup_fmt/tests/fmt.rs
---
{{^boolean}}={{/boolean}}

{{^bool}}
  * first
{{/bool}}
* {{two}}
{{^bool}}
  * third
{{/bool}}

{{^bool}}B {{^bool}}C{{/bool}} D{{/bool}}
